desafio-08/joseildoandrade12/javascript/script.js

```diff
@@ -0,0 +1,64 @@
+const readline = require("readline");
+const fs = require("fs");
+
+async function commandInline() {
+    const command = process.argv.slice(2);
+    if (command[0]) {
+        try {
+            readFileByLine(command[0]);
+        } catch (erro) {
+            console.log(erro);
+        }
+    } else {
+        console.log("Use: node script.js <caminho_do_arquivo>");
+    }
+}
+commandInline();
+
+async function readFileByLine(file) {
+    const fileStream = fs.createReadStream(file);
+    const rl = readline.createInterface({
+        input: fileStream,
+        crlfDelay: Infinity
+    });
+
+    rl.on("line", (line) => {
+        const arrayValues = valueReset(line);
+        console.log(validateValues(arrayValues));
+    });
+}
+
+function valueReset(value) {
+    return value.split("/");
+}
+
+function validateValues(value) {
+    if (value.length === 1) return value[0];
+    if ((+value[0] === 0 && +value[1] === 0) || +value[1] === 0) return "ERR";
```

Comment:
Esse if aqui não precisa ser tão grande, pode ser apenas:
```js
if (+value[1] === 0) return "ERR";
```

Entende o porquê? Se essa condição for falsa a que vem antes do `||` sempre vai ser falsa, então ela nao é necessária.